### ISSUE TITLE
Use native timer on Linux

### ISSUE THREAD

**Issue by gonX** [OpenTabletDriver/OpenTabletDriver#1815]
## Description
After #1802 was merged we are no longer using native timers on Linux.

Should they get reimplemented its important that the memory leak from #1727 is fixed.

Setting this as low priority for now as there doesn't seem to be any noticeable performance degradation on my system

**Comment by 25pwn** [comment]
Can you add a patch that doesn't use the timer while there is no input?

**Comment by gonX** [comment]
Increasing to medium priority as its probably fairly catastrophic for low core count users. 1 core is pegged at 100% for me

**Comment by AkiSakurai** [comment]
It is likely caused by regression introduced by doenet 6.0.
https://github.com/dotnet/runtime/issues/64509#issuecomment-1025140207

**Comment by jamesbt365** [comment]
#2190
